src/hotspot/share/utilities/vmError.cpp

```diff
@@ -145,13 +145,13 @@ static char* next_OnError_command(char* buf, int buflen, const char** ptr) {
 
   const char* cmd = *ptr;
 
-  // skip leading blanks or ';'
-  while (*cmd == ' ' || *cmd == ';') cmd++;
+  // skip leading blanks, ';' or newlines
+  while (*cmd == ' ' || *cmd == ';' || *cmd == '\n') cmd++;
```

Comment:
It may be worth reinstating a comment in the function description to explain how the command is actually parsed (we seem to have lost that somewhere along the line) e.g.
```
// The command string is expected to be a semi-colon, or newline, delineated sequence of commands,
// that are executed sequentially and in their own shell environment.
```
```

Reply:
Thanks David - yes, trying a new comment to introduce next_OnError_command more clearly.